src/parcsr_ls/par_cheby_device.c

```diff
@@ -0,0 +1,350 @@
+/******************************************************************************
+ * Copyright 1998-2019 Lawrence Livermore National Security, LLC and other
+ * HYPRE Project Developers. See the top-level COPYRIGHT file for details.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR MIT)
+ ******************************************************************************/
+
+/******************************************************************************
+ *
+ * Chebyshev setup and solve Device
+ *
+ *****************************************************************************/
+
+#include "_hypre_parcsr_ls.h"
+#include "_hypre_parcsr_mv.h"
+#include "float.h"
+
+#if defined(HYPRE_USING_CUDA)
+#include "_hypre_utilities.hpp"
```

Comment:
add these include's to `utilities/cuda_utils.h` if they are not there

Reply:
Resolved in  7fec6b8